backend/app/main.py

```diff
@@ -53,3 +60,47 @@ async def db_is_alive(session: Session = Depends(get_session)):
         raise HTTPException(
             status_code=status.HTTP_503_SERVICE_UNAVAILABLE, detail="DB is unreachable"
         )
+
+
+@app.post(
+    "/create_document",
+    response_model=DocumentPublic,
+    status_code=status.HTTP_201_CREATED,
+)
+async def create_document(session: Session = Depends(get_session)):
+    # To be created in the database
+    results = session.execute(text("SELECT create_document();"))
+    document_id = results.one()[0]  # should be only one row, one column of results
+    stmt = select(Document).where(Document.document_id == document_id)
+    doc = session.exec(
+        stmt
+    ).one()  # again if we've got more than one, we have problems.
+    if not doc.document_id:
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail="Document creation failed",
+        )
+
+    return doc
+
+
+@app.patch("/update_assignments")
```

Comment:
(O) for later but would love to figure out if it makes sense / is possible to remove the duplicate document_ids from `AssignmentsCreate` e.g. by using dynamic SQL